### ISSUE TITLE
Use lowercase capitalization on Featured Image "Link to..." control

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#55057]
The post type value should be lowercased, like "Link to page". 

![CleanShot 2023-10-04 at 14 16 09](https://github.com/WordPress/gutenberg/assets/1813435/e12d91ab-60f6-4ddf-aa2b-109f5ba611cd)

**Comment by properlypurple** [comment]
It looks an opposite of this was reported in #49587 and it was fixed in https://github.com/WordPress/gutenberg/pull/49591

@richtabor Is there any new discussion/decision on how this should be?

**Comment by richtabor** [comment]
It shouldn't be programmatically lowercased for sure.

**Comment by colinduwe** [comment]
The control's label is using the post type's singular_name label which is typically capitalized ([see here](https://github.com/WordPress/gutenberg/blob/86cf00a0731903b9ce6c062142adc41ebbae984a/packages/block-library/src/post-featured-image/edit.js#L156-L169))

Since the direction in https://github.com/WordPress/gutenberg/issues/49587 and from @richtabor is not to programmatically lowercase it, is this at a dead end? Or is there a lowercase singular label that I'm forgetting about?

**Comment by benoitchantre** [comment]
Per discussion with @johnbillion during the Contributor Day, the best solution would be to add a new label for post types, then use it if available. That would allow better translations in some languages.

Core ticket: https://core.trac.wordpress.org/ticket/63534

Thank you John!

**Comment by benoitchantre** [comment]
Related PR : https://github.com/WordPress/wordpress-develop/pull/8910

**Comment by johnbillion** [comment]
@richtabor What do you think about adding a new post type label as mentioned above?

**Comment by t-hamano** [comment]
Adding a new label field to the post type just for this control seems like overkill.

I'd like to suggest using the `Link to: {PostTypeLabel}` format instead:

<img width="289" height="154" alt="Image" src="https://github.com/user-attachments/assets/5f22c56f-2512-48ac-a11e-7ba938a526c7" />

This format is commonly used:

https://github.com/search?q=repo%3AWordPress%2Fgutenberg+%22%3A+%25s%27+%29%22+path%3A*.js&type=code

**Comment by Alkasingh09** [comment]
Hi, I would like to work on this issue.

**Comment by johnbillion** [comment]
I disagree that adding a new post type label is overkill, their purpose is to allow post types to provide accurate labels that read clearly and can be well translated. `Link to: Post` doesn't make much sense.

**Comment by t-hamano** [comment]
The `Link to {postType}` text currently only exists in two places, [the Post Featured Image block](https://github.com/WordPress/gutenberg/blob/bad82a0b1d5759fbe5e376163bb88837dacf93c0/packages/block-library/src/post-featured-image/edit.js#L278-L282) and [the Post Date block](https://github.com/WordPress/gutenberg/blob/bad82a0b1d5759fbe5e376163bb88837dacf93c0/packages/block-library/src/post-date/edit.js#L202-L204), so I just wanted to consider a different approach before adding new labels just for these.

However, there are no strong opinions, so if enough people prefer adding new labels, I'll obey that approach.

**Comment by t-hamano** [comment]
Another idea: change the label to `Make image a link`. The Title block has a setting called `Make title a link` to link the post. 

<img width="285" height="341" alt="Image" src="https://github.com/user-attachments/assets/fb227563-1d7e-44d9-bb98-98fe2707c359" />

**Comment by t-hamano** [comment]
Again, please note that the approach of programmatically lowercasing the post type label is not recommended - while this approach may work for English, it may not work for other locales.

**Comment by rajanarahul93** [comment]
I have a patch ready that changes the toggle label to “Make image a link,” following the suggestion from @t-hamano   
Could a maintainer confirm that this is the preferred text before I open a PR?

**Comment by t-hamano** [comment]
> I have a patch ready that changes the toggle label to “Make image a link,” following the suggestion from @t-hamano

I prefer this approach.

- It's unaffected by post types.
- It's localization-friendly.
- It doesn't require defining new post type labels.

What do others think?
